src/scripts/Providers/OSUI/Dropdown/VirtualSelect/AbstractVirtualSelect.ts

```diff
@@ -300,6 +308,9 @@ namespace Providers.OSUI.Dropdown.VirtualSelect {
 		 */
 		public close(): void {
 			this._virtualselectConfigs.close();
+
+			// Enable the global event to close on body click
+			OSFramework.OSUI.Event.GlobalEventManager.Instance.enableBodyClickEvent();
```

Comment:
Do we really need this? I think by adding the BeforeClose/Open provider events, should be enough.

Reply:
Right, but we already added a callback for any event that opens or closes the Dropdown - the BeforeClose/Open. This should fire no mather if it comes from Client Action or user interaction

Reply:
I will approve the PR anyway, but still not convinced 😂

Reply:
I removed that ;)